AmericanMedicalAssociation/ama-style-guide-2
# EWL-11080: Update homepage testimonial styling

# Pre-PR Checklist
If you have access to approved developer tools such as AMA co-pilot, ask it the following and test any suggestions before posting the PR.


## JIRA Ticket(s)
- [EWL-11087: AMAOne Design Update: Testimonials (Home Page)](https://ama-it.atlassian.net/browse/EWL-11087)


## Description:
Adjust templating and styling of testimonials on homepage.


## To Test:
- checkout ama-d8 pr locally: https://github.com/AmericanMedicalAssociation/ama-d8/pull/5170
- import latest config
- clear caches
- on homepage confirm testimonials match the following designs:
- https://www.figma.com/design/9UngoNWfohEjyomNCJ0tId/AMAOne-File-From-WM?node-id=905-1330&node-type=frame&t=B3630zpXtmbSsnHA-0
- For mobile: https://www.figma.com/design/6ZIRwcG30h03EqhR2sEVf1/Horizon-1?node-id=3597-3872&node-type=section&t=pXlcZqAwUL06p4iS-0


## Visual Regressions
Ensure any available regression testing has been run.


## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks



## Additional Notes
Anything more to add?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
